flexget/utils/search.py

```diff
@@ -11,7 +11,16 @@ def clean_symbols(text):
     result = text
     if isinstance(result, unicode):
         result = normalize('NFKD', result)
-    return re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    result = re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    
+    """Special cases"""
```

Comment:
A lot of the searches on flexget like "Kat" and "PiratBay" are giving errors because they can't find shows like Marvel's etc and Grey's Anatomy